content/en/docs/cluster-administration/storageclass.md

```diff
@@ -1,8 +1,145 @@
+| restuser | Glusterfs REST service/Heketi user who has access to create volumes in the Glusterfs Trusted Pool | 
+| secretNamespace, secretName | Identification of Secret instance that contains user password to use when talking to Gluster REST service| 
+| gidMin, gidMax | The minimum and maximum value of GID range for the StorageClass| 
+| volumetype |  The volume type and its parameters can be configured with this optional value | 
+
+For more information about StorageClass parameters, see [Glusterfs in Kubernetes Documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#glusterfs).
+
+### Ceph RBD
+Ceph RBD is also an in-tree storage plugin on Kubernetes. Volume plugin is already in Kubernetes, 
+but storage server should be installed ahead for creating storage classes of Ceph RBD.
+
+As **hyperkube** images were [deprecated since 1.17](https://github.com/kubernetes/kubernetes/pull/85094), in-tree Ceph RBD may not work without **hyperkube**.
+Nevertheless, you can use [rbd provisioner](https://github.com/kubernetes-incubator/external-storage/tree/master/ceph/rbd) as a substitute, 
+whose format is the same as in-tree Ceph RBD. The only different parameter is *provisioner* (i.e **Storage System** in KubeSphere console). 
+If you want to use rbd provisioner, the value of *provisioner* should be **ceph.com/rbd** (Input this value in **Storage System** in the image below).
+If you use in-tree Ceph RBD, the value should be *kubernetes.io/rbd*. 
+
+![Storage System](/images/storage/storage-system.png)
+
+#### Prerequisite
+- Ceph server has already installed. See [Ceph Installation Documentation](https://docs.ceph.com/en/latest/install/) for more information.
+- Install plugin if you choose to use *rbd provisioner*. Community developers provide [charts for rbd provisioner](https://github.com/kubesphere/helm-charts/tree/master/src/test/rbd-provisioner)  that you cloud use to install *rbd-provisioner* by helm. 
+
+#### Setting
+| Setting | Description |
+| :---- | :---- |  
+| monitors| Ceph monitors, comma delimited | 
+| adminId| Ceph client ID that is capable of creating images in the pool | 
+| adminSecretName| Secret Name for *adminId* | 
+| adminSecretNamespace|  The namespace for *adminSecretName* | 
+| pool | Ceph RBD pool | 
+| userId | Ceph client ID that is used to map the RBD image | 
+| userSecretName | The name of Ceph Secret for userId to map RBD image | 
+| userSecretNamespace | The namespace for *userSecretName* | 
+| fsType | fsType that is supported by kubernetes | 
+| imageFormat | Ceph RBD image format, "1" or "2" | 
+| imageFeatures| This parameter is optional and should only be used if you set imageFormat to “2”| 
+
+For more information about StorageClass parameters, see [Ceph RBD in Kubernetes Documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#ceph-rbd).
+
+### Custom StorageClass
+If none of the above storage plugins used, custom storage classes for other plugins could be chose. 
```

Comment:
->
In addition to the above storage plugins, you can also create custom storage classes of other plugins.